apps/dash-medical-provider-charges/app.py

```diff
@@ -0,0 +1,736 @@
+import dash
+import dash_table
+import dash_core_components as dcc
+import dash_html_components as html
+import plotly.graph_objs as go
+from dash.dependencies import State, Input, Output
+from dash.exceptions import PreventUpdate
+
+import pandas as pd
+import os
+
+app = dash.Dash(__name__)
+server = app.server
+
+app.config["suppress_callback_exceptions"] = True
+
+# Plotly mapbox token
+mapbox_access_token = "pk.eyJ1IjoicGxvdGx5bWFwYm94IiwiYSI6ImNqdnBvNDMyaTAxYzkzeW5ubWdpZ2VjbmMifQ.TXcBE-xg9BFdV2ocecc_7g"
+
+state_list = [
+    "AL",
+    "AZ",
+    "AR",
+    "CA",
+    "CO",
+    "CT",
+    "DC",
+    "FL",
+    "GA",
+    "IL",
+    "IN",
+    "IA",
+    "KY",
+    "MD",
+    "MA",
+    "MI",
+    "MN",
+    "MO",
+    "NE",
+    "NJ",
+    "NY",
+    "NC",
+    "OH",
+    "OK",
+    "OR",
+    "PA",
+    "SC",
+    "TN",
+    "TX",
+    "UT",
+    "VA",
+    "WA",
+    "WI",
+    "AK",
+    "DE",
+    "HI",
+    "ID",
+    "KS",
+    "LA",
+    "ME",
+    "MS",
+    "MT",
+    "NV",
+    "NH",
+    "NM",
+    "ND",
+    "RI",
+    "SD",
+    "WV",
+    "VT",
+    "WY",
+]
+
+state_map = {
+    "AK": "Alaska",
+    "AL": "Alabama",
+    "AR": "Arkansas",
+    "AZ": "Arizona",
+    "CA": "California",
+    "CO": "Colorado",
+    "CT": "Connecticut",
+    "DC": "District of Columbia",
+    "DE": "Delaware",
+    "FL": "Florida",
+    "GA": "Georgia",
+    "HI": "Hawaii",
+    "IA": "Iowa",
+    "ID": "Idaho",
+    "IL": "Illinois",
+    "IN": "Indiana",
+    "KS": "Kansas",
+    "KY": "Kentucky",
+    "LA": "Louisiana",
+    "MA": "Massachusetts",
+    "MD": "Maryland",
+    "ME": "Maine",
+    "MI": "Michigan",
+    "MN": "Minnesota",
+    "MO": "Missouri",
+    "MS": "Mississippi",
+    "MT": "Montana",
+    "NC": "North Carolina",
+    "ND": "North Dakota",
+    "NE": "Nebraska",
+    "NH": "New Hampshire",
+    "NJ": "New Jersey",
+    "NM": "New Mexico",
+    "NV": "Nevada",
+    "NY": "New York",
+    "OH": "Ohio",
+    "OK": "Oklahoma",
+    "OR": "Oregon",
+    "PA": "Pennsylvania",
+    "RI": "Rhode Island",
+    "SC": "South Carolina",
+    "SD": "South Dakota",
+    "TN": "Tennessee",
+    "TX": "Texas",
+    "UT": "Utah",
+    "VA": "Virginia",
+    "VT": "Vermont",
+    "WA": "Washington",
+    "WI": "Wisconsin",
+    "WV": "West Virginia",
+    "WY": "Wyoming",
+}
+
+# Load data
+data_dict = {}
+for state in state_list:
+    p = os.getcwd().split(os.path.sep)
+    csv_path = "data/processed/df_{}_lat_lon.csv".format(state)
+    if p[-1] != "dash-medical-provider-charges":
+        csv_path = "apps/dash-medical-provider-charges/" + csv_path
+    state_data = pd.read_csv(csv_path)
+    data_dict[state] = state_data
+
+df_al = data_dict["AL"]
```

Comment:
Are you just using this as an initial value in the graph? It might be better to leave this blank and fill in the dropdown with a default value instead. Otherwise it's not super clear why we're using AL.

Reply:
💯